google_photos_takeout_helper/__main__.py

```diff
@@ -753,4 +751,4 @@ def _walk_with_tqdm(res, bar: _tqdm):
 
 
 if __name__ == '__main__':
-    main()
+    main(sys.argv[1:])
```

Comment:
Why is that?

Reply:
to be able to call `main` from the tests and pass "command line args" as parameters.